src/qutipinterface.jl

```diff
@@ -0,0 +1,119 @@
+
+`sesolve` in QuDynamics, extended version.
+
+### Arguments
+
+Inputs :
+* H/SE/LVNE :: AbstractQuArray/QuSchrodingerEq/QuLiouvillevonNeumannEq Types
+
+  This parameter supports Hamiltonian of the system, Schrodinger Equation, LiouvillevonNeumann Equation
+* psi :: AbstractQuArray
+
+  Initial state vector or density matrix of the system
+* tlist
+
+  Time step array/range
+* solver
+
+  Keyword for selecting the solver from the various solvers supported by QuDynamics
+
+Output :
+* Array of tuples, which specify the time-step and the corresponding evolved state
+""" ->
+function sesolve(H::QuBase.AbstractQuArray, psi::QuBase.AbstractQuArray, tlist, solver=QuExpmV())
+    evolved_state = Array(Tuple, length(tlist)-1)
+    i = 1
+    for (t, psi) in QuStateEvolution(H, psi, tlist, solver)
+        evolved_state[i] = (t,psi)
+        i = i + 1
+    end
+    return evolved_state
+end
+
+sesolve(SE::QuSchrodingerEq, psi::QuBase.AbstractQuVector, tlist, solver=QuExpmV()) = sesolve(SE.hamiltonian, psi, tlist, solver)
+sesolve(LVNE::QuLiouvillevonNeumannEq, rho0::QuBase.AbstractQuMatrix, tlist, solver=QuExmpV()) =  sesolve(LVNE.liouvillian, rho0, tlist, solver)
+
+@doc """
+Interface for `mesolve`[http://qutip.org/docs/3.1.0/apidoc/functions.html#module-qutip.mesolve] of QuTiP in QuDynamics
+
+`mesolve` in QuDynamics, extended version.
+
+### Arguments
+
+Inputs :
+* H/ME :: AbstractQuArray/QuLindbladMasterEq Types
+
+  This parameter supports Hamiltonian of the system, Lindblad Master Equation
+* psi :: AbstractQuArray
+
+  Initial density matrix of the system
+* tlist
+
+  Time step array/range
+* solver
+
+  Keyword for selecting the solver from the various solvers supported by QuDynamics
+
+Output :
+* Array of tuples, which specify the time-step and the corresponding evolved state
+""" ->
+function mesolve(H::QuBase.AbstractQuArray, rho0::QuBase.AbstractQuMatrix, tlist, c_ops::Vector, solver=QuExpmV())
+    evolved_state = Array(Tuple, length(tlist)-1)
+    i = 1
+    for (t, psi) in QuStateEvolution(H, c_ops, rho0, tlist, solver)
+        evolved_state[i] = (t,psi)
+        i = i + 1
+    end
+    return evolved_state
+end
+
+mesolve(ME::QuLindbladMasterEq, rho0::QuBase.AbstractQuMatrix, tlist, solver=QuExpmV()) = mesolve(ME.hamiltonian, rho0, tlist, ME.collapse_ops, solver)
+
+@doc """
+Interface for `mcsolve`[http://qutip.org/docs/3.1.0/apidoc/functions.html#module-qutip.mcsolve] of QuTiP in QuDynamics
+
+`mcsolve` in QuDynamics, extended version.
+
+### Arguments
+
+Inputs :
+* H/ME :: AbstractQuArray/QuLindbladMasterEq Types
+
+  This parameter supports Hamiltonian of the system, Lindblad Master Equation
+* psi :: AbstractQuArray
+
+  Initial state vector of the system
+* tlist
+
+  Time step array/range
+* solver
+
+  Keyword for selecting the solver from the various solvers supported by QuDynamics
+
+Output :
+* Array of tuples, which specify the time-step and the corresponding evolved state
+""" ->
```

Comment:
Yup ! I guess I had the same doubt, but in the docs they say it is a state vector. http://qutip.org/docs/3.1.0/apidoc/functions.html#module-qutip.mcsolve. I guess we could promote this to `AbstractQuArray`.